test/integration/contributionsController.test.js

```diff
@@ -0,0 +1,155 @@
+            expect(res.body).to.be.a('object')
+            expect(res.body.message).to.equal('User added successfully!')
+            expect(res.body.userId).to.be.a('string')
+
+            return done()
+          })
+      })
+    })
+  })
+  describe('POST /tasks - creates a new task', function () {
+    it('Should return success response after adding the task (isNoteWorthy = true)', function (done) {
+      chai
+        .request(app)
+        .post('/tasks')
+        .send({
+          title: 'Task for contribution (GET contributions/:username API)',
+          purpose: 'To Test mocha',
+          featureUrl: '<testUrl>',
+          type: 'Dev | Group',
+          links: [
+            'https://github.com/Real-Dev-Squad/website-backend/pull/145'
+          ],
+          endsOn: '<unix timestamp>',
+          startedOn: '<unix timestamp>',
+          status: 'Active',
+          ownerId: '<app owner user id>',
+          percentCompleted: 10,
+          dependsOn: [
+            'Depends on task 2'
+          ],
+          participants: ['prakash'],
+          completionAward: { gold: 3, bronze: 300 },
+          lossRate: { gold: 1 },
+          isNoteworthy: true
+        })
+        .end((err, res) => {
+          if (err) { return done() }
+          expect(res).to.have.status(200)
+          expect(res.body).to.be.a('object')
+          expect(res.body.message).to.equal('Task created successfully!')
+          expect(res.body.id).to.be.a('string')
+          expect(res.body.task).to.be.a('object')
+          return done()
+        })
+    })
+    it('Should return success response after adding the task (isNoteWorthy = false)', function (done) {
+      chai
+        .request(app)
+        .post('/tasks')
+        .send({
+          title: 'Task for contribution',
+          purpose: 'To Test mocha',
+          featureUrl: '<testUrl>',
+          type: 'Dev | Group',
+          links: [
+            'https://github.com/Real-Dev-Squad/website-backend/pull/83'
+          ],
+          endsOn: '<unix timestamp>',
+          startedOn: '<unix timestamp>',
+          status: 'Active',
+          ownerId: '<app owner user id>',
+          percentCompleted: 10,
+          dependsOn: [
+            'Depends on task 2'
+          ],
+          participants: ['prakash'],
+          completionAward: { gold: 3, bronze: 300 },
+          lossRate: { gold: 1 },
+          isNoteworthy: false
+        })
+        .end((err, res) => {
+          if (err) { return done() }
+          expect(res).to.have.status(200)
+          expect(res.body).to.be.a('object')
+          expect(res.body.message).to.equal('Task created successfully!')
+          expect(res.body.id).to.be.a('string')
+          expect(res.body.task).to.be.a('object')
+          return done()
+        })
+    })
+  })
+  describe('GET /contributions/{username}', function () {
+    it('Should get all the contributions of the user', function (done) {
```

Comment:
https://github.com/Real-Dev-Squad/website-backend/blob/develop/routes/contributions.js#L19 shows that this route also has 404, 500 responses. Can you also add tests for that?